cmd/go-tpc/tpch.go

```diff
@@ -29,8 +30,13 @@ var queryTuningVars = []struct {
 
 // isSysVarSupported determines if a system variable is supported in given TiDB version
 // TODO: Every known sys var should have a minimal supported version and be checked individually. For now we just assume all sys vars are supported since 7.1.0.
-func isSysVarSupported(ver util.SemVersion, sysVar string) bool {
-	return ver.Compare(util.SemVersion{Major: 7, Minor: 1, Patch: 0}) >= 0
+func isSysVarSupported(ver *semver.Version, _ string) bool {
+	return ver.Compare(semver.Version{Major: 7, Minor: 1, Patch: 0}) >= 0
+}
+
+// isInvertedIndexSupported checks if inverted index is supported in given TiDB version
+func isInvertedIndexSupported(ver semver.Version) bool {
+	return ver.Compare(semver.Version{Major: 9, Minor: 0, Patch: 0, PreRelease: "beta.1"}) > 0
```

Comment:
Does `9.0.0-beta.2 > 9.0.0-beta.1 == true`?

Reply:
Yes, check the UT `TestSemVersionCompare`